src/VisualStudio/Core/Def/Implementation/Diagnostics/VisualStudioDiagnosticAnalyzerProvider.WorkspaceEventListener.cs

```diff
@@ -21,10 +24,10 @@ namespace Microsoft.VisualStudio.LanguageServices.Implementation.Diagnostics
     internal partial class VisualStudioDiagnosticAnalyzerProvider
     {
         /// <summary>
-        /// Loads VSIX analyzers into <see cref="VisualStudioWorkspace"/> when it's loaded.
+        /// Loads VSIX analyzers into workspaces that provide <see cref="ISolutionAnalyzerSetterWorkspaceService"/> when they are loaded.
         /// </summary>
         [Export]
-        [ExportEventListener(WellKnownEventListeners.Workspace, WorkspaceKind.Host), Shared]
+        [ExportEventListener(WellKnownEventListeners.Workspace, WorkspaceKind.Host, WorkspaceKind.Interactive), Shared]
```

Comment:
I guess my imagination at this point was we'd just export this for all workspace kinds and just rely on the service existence. I guess that doesn't work though. :-/